commands/Factorio/linkme.js

```diff
@@ -0,0 +1,67 @@
+const { MessageEmbed } = require("discord.js");
+const Command = require("../../base/Command.js");
+
+class Linkme extends Command {
+  constructor(client) {
+    super(client, {
+      name: "linkme",
+      description: "Link yourself",
```

Comment:
This might make more sense if it said `link your factorio and discord accounts!`